src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -569,7 +569,7 @@ See the accompanying LICENSE file for applicable license.
       <xsl:apply-templates/>
       <xsl:choose>
         <xsl:when test="@href">
-          <br/><div style="text-align:right"><a>
+          <br/><div><a>
```

Comment:
There's one more on line 588.

Reply:
@jelovirt Thanks, good catch. Resolved in de7542f.